source/android/adaptivecards/src/androidTest/java/io/adaptivecards/objectmodel/ChoiceSetInputPropertiesTest.java

```diff
@@ -211,6 +229,72 @@ public ChoiceSetInput castTo(BaseCardElement element) {
         }
     }
 
+    @Test
+    public void isRequiredCompactValidation()
+    {
+        ChoiceSetInput choiceSetInput = TestUtil.createMockChoiceSetInput();
+        choiceSetInput.SetIsRequired(true);
+
+        ComboBoxInputHandler choiceSetInputHandler = new ComboBoxInputHandler(choiceSetInput);
+        Spinner spinner = new Spinner(InstrumentationRegistry.getContext());
+        spinner.setAdapter(new ArrayAdapter<String>(InstrumentationRegistry.getContext(), android.R.layout.simple_spinner_item, new String[]{"sample title", ""}));
+        spinner.setSelection(1);
+        choiceSetInputHandler.setView(spinner);
+
+        TestUtil.GeneralValidationExecutor gralExecutor = new TestUtil.GeneralValidationExecutor(choiceSetInputHandler);
```

Comment:
Let's try to follow good naming guidelines even for tests (Treat test code as prod code). "generalExecutor"